class-instant-articles-post.php

```diff
@@ -899,20 +899,29 @@ public function should_submit_post() {
 			return false;
 		}
 
-		// Transform the post to an Instant Article.
-		$adapter = new Instant_Articles_Post( $post );
+		// Don't publish posts with password protection
+		if ( post_password_required( $post ) ) {
+			return false;
+		}
+
+		// Allow to disable post submit via filter
+		if ( false === apply_filters( 'instant_articles_should_submit_post', true, $this ) ) {
+			return false;
+		}
+
+		$cache = get_post_meta( $this->get_the_id(), 'should_submit_post_content', true );
```

Comment:
This post meta is only used internally by the plugin, it is not on the public interface, so I'll add in-code documentation.

There are no filters being added by this PR.